### PR TITLE
feat: Add `appMetadata` when initializing CozyClient

### PR DESCRIPTION
**Pull request by edas**
Adding the `appMetadata` object to CozyClient will allow the later to
take care of the `cozyMetadata` fields in couch documents.

### CODE REVIEW

File: packages/cozy-scripts/template/app/src/targets/browser/index.jsx [hunk 1]
```diff
@@ -50,6 +50,13 @@ document.addEventListener('DOMContentLoaded', () => {
     require('../../../manifest.webapp').name
   )
 
+  const appSlug = getDataOrDefault(
+    data.cozyAppSlug,
+    require('../../../manifest.webapp').slug
+  )
+
+  const appVersion = require('../../../manifest.webapp').version
```
Comment (drazik) [inline packages/cozy-scripts/template/app/src/targets/browser/index.jsx]: Can we `require('../../..manifest.webapp')` only one time at the top of the file?

Comment (edas) [reply packages/cozy-scripts/template/app/src/targets/browser/index.jsx]: done

**Comment by edas** [comment]
J'ai des erreurs de test, malheureusement reproductible sur la branche master actuelle, donc pas liée à mes modif. Je remets l'investigation à plus tard

**Comment by nono** [comment]
It looks like there are a lot of files in conflict. Should we close this PR?

**Comment by ptbrowne** [comment]
Yep, I do not think we will be able to use this branch.

**Pull request closed without merging**